docker/foxpass_sg_update_tool.py

```diff
@@ -73,6 +75,21 @@ def main():
     update_security_group(add_ports, clear_ports, rules, security_group, task_name)
 
 
+# In Lambda run this instead of main()
+def lambda_handler(event, context):
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+    ec2 = boto3.resource('ec2')
+    ecs = boto3.client('ecs')
+    cluster, sg_id, task_name = (os.environ.get('CLUSTER'), os.environ.get('SG_ID'), os.environ.get('TASK_NAME'))
+    security_group = ec2.SecurityGroup(sg_id)
+    active_ports = get_ports(cluster, ecs, task_name)
+    rules = get_current_sg_rules(security_group, task_name)
+    add_ports, clear_ports = ports_to_change(active_ports, rules)
+    update_security_group(add_ports, clear_ports, rules, security_group, task_name)
+    logger.info('Active: {}\tAdded: {}\tCleared: {}'.format(active_ports, add_ports, clear_ports))
+
```

Comment:
if this function throws an exception, does lambda catch it and show it to us somewhere?